qa/L0_http/generate_models/mock_llm/1/model.py

```diff
@@ -87,7 +87,10 @@ def exec_decoupled(self, requests):
                 for _ in range(rep_count):
                     if delay is not None:
                         time.sleep(delay)
-                    sender.send(response)
+                    if not sender.is_cancelled():
+                        sender.send(response)
```

Comment:
Does the server segfault if we send responses when the request is cancelled and connection is closed?

Reply:
no - server can handle if responses are sent. this was changed to showcase the feature